Tests/Extensions/ArrayExtensionTests.swift

```diff
@@ -73,4 +73,11 @@ class ArrayExtensionTests: XCTestCase {
             XCTAssertEqual(testArray[index], sortedArray[index])
         }
     }
+    
+    func testDistinct() {
+        let testArray = [0, 1, 2, 2, 2, 3, 4, 5, 6, 6]
+        let resultArray = [0, 1, 2, 3, 4, 5, 6]
+        
```

Comment:
Here, tests for edge cases like empty arrays would be important. But according to my above comments, this should be removed from this PR anyways – but for the future...